<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CCanvasSettingPanel</class>
 <widget class="QWidget" name="CCanvasSettingPanel">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>282</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Settings</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3" stretch="0,0,0">
   <property name="spacing">
    <number>3</number>
   </property>
   <property name="margin">
    <number>4</number>
   </property>
   <item>
    <widget class="QWidget" name="m_stylesHolder" native="true">
     <layout class="QVBoxLayout" name="verticalLayout" stretch="0,1,0">
      <property name="margin">
       <number>0</number>
      </property>
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Background Style (Click to set)</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QListView" name="m_styleList">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>0</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="widget_3" native="true">
        <layout class="QHBoxLayout" name="horizontalLayout" stretch="0,1,0">
         <property name="spacing">
          <number>0</number>
         </property>
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>5</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QCheckBox" name="m_setAsDefaultStyle">
           <property name="text">
            <string>Set as Default Style</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QWidget" name="widget_4" native="true"/>
         </item>
         <item>
          <widget class="QToolButton" name="m_btnMoreStyle">
           <property name="text">
            <string>More...</string>
           </property>
           <property name="autoRaise">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="Line" name="line">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="widget_2" native="true">
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <property name="margin">
       <number>0</number>
      </property>
      <item>
       <widget class="QCheckBox" name="m_showGrids">
        <property name="text">
         <string>Enable Grids</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
